LICENSE

```diff
@@ -2,15 +2,15 @@
   LICENSE ISSUES
   ==============
 
-  The OpenSSL toolkit stays under a dual license, i.e. both the conditions of
+  The OpenSSL toolkit stays under a double license, i.e. both the conditions of
```

Comment:
Right, but we are using it wrongly.  Dual as usually used means "a or b"  Ours is double, meaning "a and b"